Packs/HelloWorld/Integrations/HelloWorldX2/HelloWorldX2.py

```diff
@@ -0,0 +1,1478 @@
+        # On the other hand, this API is way to verbose, so we want to select
+        # only certain keys to be returned in order not to clog the context
+        # with useless information. What to actually return in the context and
+        # to define as a command output is subject to design considerations.
+
+        # INTEGRATION DEVELOPER TIP
+        # To generate the Context Outputs on the YML use ``demisto-sdk``'s
+        # ``json-to-outputs`` option.
+
+        # Define which fields we want to exclude from the context output as
+        # they are too verbose.
+        ip_context_excluded_fields = ['objects', 'nir']
+        ip_data = {k: ip_data[k] for k in ip_data if k not in ip_context_excluded_fields}
+
+        # In this case we want to use an custom markdown to specify the table title,
+        # but otherwise ``CommandResults()`` will call ``tableToMarkdown()``
+        #  automatically
+        readable_output = tableToMarkdown('IP', ip_data)
+
+        # INTEGRATION DEVELOPER TIP
+        # The output key will be ``HelloWorld.IP``, using ``ip`` as the key field.
+        # ``indicator`` is used to provide the context standard (IP)
+        command_results.append(CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='HelloWorld.IP',
+            outputs_key_field='ip',
+            outputs=ip_data,
+            indicator=ip_standard_context,
+            relationships=relationships_list
+        ))
+    return command_results
+
+
+def domain_reputation_command(client: Client, args: Dict[str, Any], default_threshold: int) -> List[CommandResults]:
+    """domain command: Returns domain reputation for a list of domains
+
+    :type client: ``Client``
+    :param Client: HelloWorld client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['domain']`` list of domains or a single domain
+        ``args['threshold']`` threshold to determine whether a domain is malicious
+
+    :type default_threshold: ``int``
+    :param default_threshold:
+        default threshold to determine whether an domain is malicious
+        if threshold is not specified in the XSOAR arguments
+
+    :return:
+        A ``CommandResults`` object that is then passed to ``return_results``,
+        that contains Domains
+
+    :rtype: ``CommandResults``
+    """
+
+    # INTEGRATION DEVELOPER TIP
+    # Reputation commands usually support multiple inputs (i.e. arrays), so
+    # they can be invoked once in XSOAR. In this case the API supports a single
+    # IP at a time, so we will cycle this for all the members of the array.
+    # We use argToList(), implemented in CommonServerPython.py to automatically
+    # return a list of a single element even if the provided input is a scalar.
+
+    domains = argToList(args.get('domain'))
+    if len(domains) == 0:
+        raise ValueError('domain(s) not specified')
```

Comment:
The statement is redundant as the `domain` argument cannot be empty.
```suggestion
    if not domains:
        raise ValueError('domain(s) not specified')
```